### ISSUE TITLE
build failure: undeclared function 'T238f30e1'

### ISSUE THREAD

**Issue by mw66** [gobo-eiffel/gobo#75]
In this branch:

https://github.com/joortcom/gobo/tree/detect_diamond

```
/gobo/tool/gedoc$ make gobo 
gec --gc=no --capability=concurrency=none   src/system.ecf
Degree 6: 0/0/0 0:0:0.333
Degree 5: 0/0/0 0:0:0.922
Degree 4: 0/0/0 0:0:0.265
Degree 3: 0/0/0 0:0:0.260
Degree -2: 0/0/0 0:0:1.908
Degree -3: 0/0/0 0:0:2.182
gedoc7.c:99214:10: error: call to undeclared function 'T238f30e1'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
 99214 |                                 t5 = T238f30e1(ac, l12, a1);
       |                                      ^
1 error generated.
Degree -4: 0/0/0 0:0:9.689
Total Time: 0/0/0 0:0:15.560
make: *** [Makefile:13: gobo] Error 1

/gobo/tool/gedoc$ grep -rwI T238f30e1 *
gedoc7.c:				t5 = T238f30e1(ac, l12, a1);

```

**Comment by ebezault** [comment]
This issue is fixed in 902c6ab58b5cd98d439c8a7d78411e6982cfca23.

**Comment by mw66** [comment]
Fix verified.
